src/libraries/System.IO.Compression/src/System/IO/Compression/ZipBlocks.cs

```diff
@@ -482,44 +482,50 @@ public static bool TryValidateBlock(BinaryReader reader, ZipArchiveEntry entry)
             }
             else
             {
-                if (reader.BaseStream.Length < reader.BaseStream.Position + extraFieldLength + entry.CompressedLength + 4)
+                if (reader.BaseStream.Length < reader.BaseStream.Position + extraFieldLength + entry.CompressedLength + 12)
```

Comment:
Can the magic number 12 be explained in a comment or made a const with an explanatory name?

Reply:
Sure